### PR TITLE
Bau/send code to backend

### PR DESCRIPTION
**Pull request by jkunle**
### What changed

This is a frontend only implementation of sending the authorization code to the backend.

It is currently failing in integration, and it also not currently working on PaaS

### Issue tracking

- [PYI-373](https://govukverify.atlassian.net/browse/PYI-373)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

**Review by merlinc — DISMISSED** [review]
This is good enough for us to merge and investigate why things are broken - we can come back to any stylistic things once everything is working again.

**Comment by kerrr** [comment]
Please can we have the PR comment make some kind of sense. This is just the template!

If we just need a give something a kick then we can strip out the template and write what the purpose is.

**Comment by kerrr** [comment]
And does it really not relate to any ticket?

**Comment by merlinc** [comment]
@kerrr We're trying to unbreak the deployments (PaaS is broken too) so we're lowering the quality gates for an hour or so. I'm updating the description with some information to say that.

File: .nycrc.yml [hunk 1]
```diff
@@ -5,10 +5,10 @@ include:
 exclude:
   - "**/*.test.js"
 report-dir: coverage
-statements: 40
-branches: 90
-functions: 50
-lines: 40
+statements: 10
```
Comment (merlinc) [inline .nycrc.yml:8]: These will be restored later.

File: src/app/credential-issuer/middleware.test.js [hunk 2]
```diff
@@ -130,4 +134,62 @@ describe("credential issuer middleware", () => {
...
+        axios: axiosStub,
+        "../../lib/config": configStub,
+      });
+
+      await middleware.sendParamsToAPI(req, res, next);
+
+      expect(res.status).to.be.eql(200);
+    });
+
+    it.skip("should send code to core backend and return with a 404 response", async () => {
```
Comment (merlinc) [inline src/app/credential-issuer/middleware.test.js:174]: This will be fixed later.

**Review by merlinc — APPROVED** [review]

**Pull request merged**